bedrock/firefox/templates/firefox/features/picture-in-picture.html

```diff
@@ -0,0 +1,112 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% from "macros-protocol.html" import hero with context %}
+
+{% extends "firefox/base/base-protocol.html" %}
+
+{% block page_title %}{{ ftl('features-pip-firefox-picture-in-picture-get-more') }}{% endblock %}
+{% block page_desc %}{{ ftl('features-pip-got-things-to-do-and-things-to-watch') }}{% endblock %}
+{% block page_image %}{{ static('img/firefox/features/pip/pip-hero.jpg') }}{% endblock %}
+
+{% block page_css %}
+  {{ css_bundle('firefox-features-pip') }}
+{% endblock %}
+
+{% block sub_navigation %}
+  {% include "firefox/features/includes/subnav.html" %}
+{% endblock %}
```

Comment:
Quesiton: Should the PiP page have a sub-nav item? I'm only seeing three pages listed. 

<img width="1526" alt="image" src="https://user-images.githubusercontent.com/2692333/109689181-a884bd80-7b4a-11eb-8544-904cf023a338.png">

<img width="506" alt="image" src="https://user-images.githubusercontent.com/2692333/109689193-ab7fae00-7b4a-11eb-8058-4e846e0f9a9b.png">

Reply:
These were the three we chose to highlight when we first added the subnav. I'd vote to leave the subnav alone for now but we can ask Raphael about adding PiP later, or maybe making other changes.

Reply:
Yeah. There's definitely an experiment to be had on which items should be in here. (if not already!)